docs/admin/nondefault-agent-socket.txt

```diff
@@ -0,0 +1,2 @@
+DAOS_AGENT_DRPC_DIR
```

Comment:
Should we not learn here that this is the env variable to be used with a new patch if man want to change the default directory for the Agent's socket ??

Reply:
that was a reminder to myself to add documentation for that (in a separate PR), which accidentally got committed here...